content/api/rest-sdk/types.textile

```diff
@@ -15,6 +15,7 @@ languages:
   - objc
   - csharp
   - go
+  - flutter
```

Comment:
This is the right change, but results in `ErrorInfo` being blank for Flutter (see [here](https://ably-docs-edu-61-broken-whlym6.herokuapp.com/docs/api/rest-sdk/types?lang=flutter#error-info)). 

I think you also need to add Flutter to the `blang` that pulls in the `ErrorInfo` partial on L83.

Reply:
I'm still seeing `ErrorInfo` as blank for Flutter with this change. We might have to catch up and try a few things to fix this, as there's clearly a bug somewhere.

Reply:
Hmm 💭 🤔 @mark ok 👍

Reply:
> I'm still seeing `ErrorInfo` as blank for Flutter with this change. We might have to catch up and try a few things to fix this, as there's clearly a bug somewhere.

@m-hulbert, please see fixup! b366b3499862926e638adeb203eb1782f30bb9c2